documentation/modules/cruise-control/proc-deploying-cruise-control.adoc

```diff
@@ -0,0 +1,106 @@
++
+[source,yaml,subs="attributes+"]
+----
+apiVersion: {KafkaApiVersion}
+kind: Kafka
+metadata:
+  name: my-cluster
+spec:
+  # ...
+  cruiseControl:
+    replicas: 1 <1>
+    capacity: <2>
+      networkIn: 10000KB/s
+      networkOut: 10000KB/s
+      # ...
+    config: <3>
+      default.goals: >
+         com.linkedin.kafka.cruisecontrol.analyzer.goals.RackAwareGoal,
+         com.linkedin.kafka.cruisecontrol.analyzer.goals.ReplicaCapacityGoal
+      cpu.balance.threshold: 1.1
+      metadata.max.age.ms: 300000
+      send.buffer.bytes: 131072
+      # ...
+    resources: <4>
+      requests:
+        cpu: 200m
+        memory: 64Mi
+      limits:
+        cpu: 500m
+        memory: 128Mi
+    logging: <5>
+        type: inline
+        loggers:
+          cruisecontrol.root.logger: "INFO"
+    template: <6>
+      pod:
+        metadata:
+          labels:
+            label1: value1
+        securityContext:
+          runAsUser: 1000001
+          fsGroup: 0
+        terminationGracePeriodSeconds: 120
+    readinessProbe: <7>
+      initialDelaySeconds: 15
+      timeoutSeconds: 5
+    livenessProbe: <8>
+      initialDelaySeconds: 15
+      timeoutSeconds: 5
+# ...
+----
+<1> Number of Cruise Control instances to create. Set to `1` to run the application or `0` to pause it.
+<2> Specifies capacity limits for broker resources. For more information, see xref:capacity_configuration[Capacity configuration].
+<3> Defines the Cruise Control configuration, including default and enabled optimization goals. You can provide any xref:ref-cruise-control-configuration-{context}[standard configuration option] apart from those managed directly by {ProductName}.
+<4> CPU and memory resources reserved for Cruise Control. For more information, see xref:assembly-resource-limits-and-requests-deployment-configuration-kafka[].
+<5> Defined loggers and log levels added directly (inline) or indirectly (external) through a ConfigMap. A custom ConfigMap must be placed under the log4j.properties key. Cruise Control has a single logger called cruisecontrol.root.logger. You can set the log level to INFO, ERROR, WARN, TRACE, DEBUG, FATAL or OFF. For more information, see xref:logging_configuration[Logging configuration].
+<6> xref:assembly-customizing-deployments-str[Customization of deployment templates and pods].
+<7> xref:assembly-healthchecks-deployment-configuration-kafka[Healthcheck readiness probes].
+<8> xref:assembly-healthchecks-deployment-configuration-kafka[Healthcheck liveness probes].
+
+. Create or update the resource:
++
+[source,shell,subs="+quotes"]
+----
+kubectl apply -f _kafka.yaml_
+----
+
+. Verify that Cruise Control was successfully deployed:
++
+[source,shell,subs="+quotes"]
+----
+kubectl get deployments -l app.kubernetes.io/name=strimzi
+----
+
+.What to do next
+
+After configuring and deploying Cruise Control, you can xref:proc-interacting-with-cruise-control-api-{context}[interact with the Cruise Control API]
```

Comment:
@kyguy - Please comment this out because the target procedure is not ready. Also, I think the best next step is to generate optimization proposals and use them to rebalance the cluster.